src/main/kotlin/dsl/Person.kt

```diff
@@ -0,0 +1,37 @@
+package dsl
+
+data class Person(
+    var name: String,
+    var company: String? = null,
+    val skills: Skills = Skills(),
+    val languages: Languages = Languages(),
+)
+
+class PersonBuilder {
+    private lateinit var name: String
+    private var company: String? = null
+    private val skills = Skills()
+    private val languages = Languages()
```

Comment:
Do you know the `@DslMarker` annotation? If not, you should study it. Also, could you tell me the advantages of using `@DslMarker` when you have time?